src/main/java/de/embl/cba/mobie/image/ImagePropertiesToMetadataAdapter.java

```diff
@@ -32,4 +34,21 @@ public void setMetadata( Metadata metadata, MutableImageProperties imageProperti
 		metadata.showSelectedSegmentsIn3d = imageProperties.showSelectedSegmentsIn3d;
 		metadata.additionalSegmentTableNames = imageProperties.tables;
 	}
+
+	public void setImageProperties( Metadata metadata, MutableImageProperties imageProperties) {
```

Comment:
rename to `setMutableImagePropertiesFromMetadata` and change the order of the arguments:

`setMutableImagePropertiesFromMetadata( MutableImageProperties properties, Metadata metadata )`

My above method `setMetadata` should be renamed accordingly:

`setMetadataFromMutableImageProperties`